<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BreakpointViewer</class>
 <widget class="QTabWidget" name="BreakpointViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>150</height>
   </size>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tabBpLabel">
   <attribute name="title">
    <string>Breakpoints</string>
   </attribute>
   <layout class="QGridLayout" name="_19">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="_20">
      <item>
       <widget class="QPushButton" name="btnAddBp">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemoveBp">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTableWidget" name="bpTableWidget">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <column>
       <property name="text">
        <string>Enabled</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Condition</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Command</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Once</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabWpLabel">
   <attribute name="title">
    <string>Watchpoints</string>
   </attribute>
   <layout class="QGridLayout" name="_21">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="_22">
      <item>
       <widget class="QPushButton" name="btnAddWp">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemoveWp">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDisableWp">
        <property name="text">
         <string>Disable</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTableWidget" name="wpTableWidget">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="rootIsDecorated" stdset="0">
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights" stdset="0">
       <bool>true</bool>
      </property>
      <property name="itemsExpandable" stdset="0">
       <bool>false</bool>
      </property>
      <property name="allColumnsShowFocus" stdset="0">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Enabled</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Region</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Condition</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Command</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
